src/common/cockpitwebresponse.c

```diff
@@ -1623,6 +1626,7 @@ cockpit_web_response_negotiation (const gchar *path,
   gchar *shorter = NULL;
   gchar *lang = NULL;
   gchar *lang_region = NULL;
+  gboolean _is_lang, _is_compressed;
```

Comment:
ugh.

If you want to do this, the usual convention is rather to use `out_is_language_specific` as the function parameter and call the variable `is_language_specific`.